src/pveclib/vec_f128_ppc.h

```diff
@@ -1263,6 +1963,51 @@ static inline __binary128 vec_xsiexpqp (vui128_t sig, vui64_t exp);
    return result;
  }
 
+ /** \brief Transfer a quadword from a __binary128 scalar to a vector int
+  * and logical AND Compliment with mask.
```

Comment:
s/logical AND with mask/Logical AND with Mask/

Reply:
Actually this is a pasto, It should be:
/and logical Exclusive OR with mask/